lab-isaiah/lib/arithmetic.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+let arithmetic = module.exports = {};
+
+arithmetic.add = (a, b) => {
+  let num1 = Number(a);
```

Comment:
This is okay for this context (but would not work if you had tested for the bonus points), so rather than converting the input to a number, you should validate that the type of input is a number and if it is not, return null